workshop/Lab1/README.md

```diff
@@ -0,0 +1,97 @@
+# Lab 1: Deploy an application with in-memory storage
+
+In this Lab we'll learn how to deploy an application hosted within the IBM Kubernetes Service that stores
+state in memory.
+
+# Deploy your application
+
+In this part of the lab we will deploy an application called `guestbook`
+that has already been built and uploaded to DockerHub under the name
+`ibmcom/guestbook:v1`.
+
+Start by running `guestbook`:
+
+```
+$ kubectl run guestbook --image=ibmcom/guestbook:v1
+```
+
+This action will take a bit of time. To check the status of the running application,
+you can use `kubectl get pods`.
+
+You should see output similar to the following:
+```console
+$ kubectl get pods
+NAME                          READY     STATUS              RESTARTS   AGE
+guestbook-59bd679fdc-bxdg7    0/1       ContainerCreating   0          1m
+```
+Eventually, the status should show up as `Running`.
+```console
+$ kubectl get pods
+NAME                          READY     STATUS              RESTARTS   AGE
+guestbook-59bd679fdc-bxdg7    1/1       Running             0          1m
+```
+
+The end result of the run command is not just the pod containing our application containers,
+but a Deployment resource that manages the lifecycle of those pods.
```

Comment:
You may want to expand on this thought a bit since Kube itself does this w/o the need for a Deployment. So perhaps adding something about how a Deployment goes beyond what a normal pod spec does, which can ensure that one instance is already running, by allowing you to scale it so you can have any number of instances running.